chat_client/src/main.py

```diff
@@ -0,0 +1,162 @@
+import logging
+
+import grpc
+import protobufs.chat_pb2 as chat_pb2
+import protobufs.chat_pb2_grpc as chat_pb2_grpc
+from google.protobuf.timestamp_pb2 import Timestamp
+
+import chat_receiver
+
+# import getpass
+
+
+class ChatClient():
+    def __init__(self, host: str, port: int) -> None:
+        self._is_connected = False
+
+        self._channel = None
+        self._stub = None
+        self._receiver = None
+
+        self._login = ""
+        self._connection_addr = f"{host}:{str(port)}"
```

Comment:
Why do we need `f"{str(port)}"`?
Format string alreadt calls __str__ magic method.